tests/async/async_ref_test.cc

```diff
@@ -536,24 +419,27 @@ TEST(AsyncRef, TryAction) {
 
 	ecsact_async_enqueue_execution_options(options.c());
 
-	auto evc = ecsact_execution_events_collector{};
-	evc.entity_created_callback = entity_cb;
-	evc.entity_created_callback_user_data = &cb_info;
+	auto evc = ecsact::core::execution_events_collector{};
 
-	auto start_tick = ecsact_async_get_current_tick();
-	while(cb_info.wait != true) {
-		std::this_thread::yield();
-		ecsact_async_flush_events(&evc, nullptr);
-		auto current_tick = ecsact_async_get_current_tick();
-		auto tick_diff = current_tick - start_tick;
-		ASSERT_LT(tick_diff, 10);
-	}
+	static auto entity = ecsact_entity_id{};
```

Comment:
```suggestion
	auto entity = ecsact_entity_id{};
```
This no longer needs to be static since you're capturing now

Reply:
oops! I was wrong. It's used in the system impl. My bad!